src/view/com/pager/PagerWithHeader.tsx

```diff
@@ -114,7 +114,9 @@ export const PagerWithHeader = React.forwardRef<PagerRef, PagerWithHeaderProps>(
       (scrollRef: AnimatedRef<any> | null, atIndex: number) => {
         scrollRefs.modify(refs => {
           'worklet'
-          refs[atIndex] = scrollRef
+          if (scrollRef != null) {
+            refs[atIndex] = scrollRef
+          }
           return refs
```

Comment:
this is one to look at. haven't looked into what actually changed here yet

Reply:
Hmm I think we'd actually want to properly clear these.

Reply:
fixed differently

Reply:
ty for looking at that! wasn't really sure what changed here 🙏